### PR TITLE
Update README.md

### LINKED ISSUE #3: README needs more detail and installation/build instructions
Use https://github.com/DACCS-Climate/marble-website/blob/main/README.md as a starting point

### PR DESCRIPTION
**Pull request by alexandercyu**
Added description of the repo and how to build and run the site
Resolves #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
-Information site about the DACCS project
+# The DACCS Project Website
+This site is to give a summary of what the DACCS project is about.  It includes a mission statement, vision statement, a list of the Principle Investigators and a list of the Funders.
+
+# Building the site
+
+Clone the repository at [https://github.com/DACCS-Climate/daccs-project-website](https://github.com/DACCS-Climate/daccs-project-website)
+
+Navigate to the root directory and build the site by running 
+```angular2html
```
Comment (mishaschwartz) [inline README.md]: shell, not angular (and same below)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
-# daccs-project-website
-Information site about the DACCS project
+# The DACCS Project Website
+This site is to give a summary of what the DACCS project is about.  It includes a mission statement, vision statement, a list of the Principle Investigators and a list of the Funders.
+
+# Building the site
+
+Clone the repository at [https://github.com/DACCS-Climate/daccs-project-website](https://github.com/DACCS-Climate/daccs-project-website)
```
Comment (mishaschwartz) [inline README.md]: Not needed, they're already on that page

File: README.md [hunk 3]
```diff
@@ -1,2 +1,18 @@
...
+
+Navigate to the root directory and build the site by running 
+```angular2html
+python build.py
```
Comment (mishaschwartz) [inline README.md]: `python3` 

`python` is python 2 on most distros

(and same below)

File: README.md [hunk 4]
```diff
@@ -1,2 +1,18 @@
...
+```
+
+The site files will be automatically built and moved to the `/build` folder.
```
Comment (mishaschwartz) [inline README.md:11]: Add information about specifying a different build location. You can just use the same line from the marble-website's README.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,18 @@
-# daccs-project-website
-Information site about the DACCS project
+# The DACCS Project Website
+This site is to give a summary of what the DACCS project is about.  It includes a mission statement, vision statement, a list of the Principle Investigators and a list of the Funders.
+
+# Building the site
+
+Clone the repository at [https://github.com/DACCS-Climate/daccs-project-website](https://github.com/DACCS-Climate/daccs-project-website)
+
+Navigate to the root directory and build the site by running
```
Comment (mishaschwartz) [inline README.md]: Add a section about installing dependencies

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Why not just copy and paste everything from the marble-website's README and remove/change the information that is not relevant to this project?

As mentioned in #3: please use that README as a starting point

File: README.md (hunk 1 above)
Comment (alexandercyu) [reply README.md]: Replaced with shell

File: README.md (hunk 2 above)
Comment (alexandercyu) [reply README.md]: Removed URL

File: README.md (hunk 3 above)
Comment (alexandercyu) [reply README.md]: Changed to python3

File: README.md (hunk 4 above)
Comment (alexandercyu) [reply README.md:11]: Added info about different build directory

File: README.md (hunk 5 above)
Comment (alexandercyu) [reply README.md]: Added part about installing dependencies

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**